homeassistant/components/bond/config_flow.py

```diff
@@ -56,7 +57,30 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
 
-    _discovered: dict = None
+    def __init__(self):
+        """Initialize config flow."""
+        self._discovered: dict = None
```

Comment:
The typing should be `Optional[dict]`.

Reply:
Thanks. Will fix tomorrow after sleep

Reply:
https://github.com/home-assistant/core/pull/47055